src/search-manager/data/api.ts

```diff
@@ -121,16 +121,44 @@ export interface ContentHit {
   lastPublished: number | null;
 }
 
+/**
+ * Information about a single collection returned in the search results
+ * Defined in edx-platform/openedx/core/djangoapps/content/search/documents.py
+ */
+export interface CollectionHit {
+  id: string;
+  type: 'collection';
```

Comment:
I'm not sure it's useful to have a separate `CollectionHit` type, when the `ContentHit` type has all the same fields? You can just defined `ContentHit.type` to be `'course_block' | 'library_block' | 'collection'`

Or you can have a ContentHit base interface and BlockHit can extend it to add the `usageKey` and `content` fields, while `CollectionHit` can extend it to add the (future) `componentCount` field (https://github.com/openedx/frontend-app-course-authoring/issues/1260).

Reply:
Actually, not all fields are same. Created a base content hit interface and extended it in both types.